src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -122,18 +122,20 @@ public CaseResponse transformExceptionRecord(ExceptionRecord exceptionRecord, bo
             return formTypeValidatorResponse;
         }
 
-        String formType = exceptionRecord.getFormType();
-        log.info("formtype for case {} is {}", caseId, formType);
 
-        if (formType == null || notAValidFormType(formType)) {
-            ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
-            String ocrFormType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+        ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
+        String formType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+
+        if (formType == null  || notAValidFormType(formType)) {
+            formType = exceptionRecord.getFormType();
 
-            if (!notAValidFormType(ocrFormType)) {
-                formType = ocrFormType;
+            if (formType == null || notAValidFormType(formType)) {
+                formType = null;
             }
         }
 
+        log.info("formtype for case {} is {}", caseId, formType);
+
```

Comment:
This is duplicating the method getFormType(), can you use that public method instead?

Reply:
I tried that to group into one method, but it will affect lot of test case result and not worth to change it